arbitrum-docs/arbos/l1-to-l2-messaging.mdx

```diff
@@ -1,64 +1,86 @@
+
+- If the ticket is sucessfully auto-redeemed, it will execute with the sender, destination, callvalue, and calldata of the original submission. The submission fee is refunded to the user on L2 (`excessFeeRefundAddress`). Note that to ensure successful auto-redeem of the ticket, one could use the Arbitrum SDK which provides a [convenience function][https://github.com/OffchainLabs/arbitrum-sdk/blob/4cedb1fcf1c7302a4c3d0f8e75fb33d82bc8338d/src/lib/message/L1ToL2MessageGasEstimator.ts#L215] that returns the desired gas parameters when sending L2-L1 messages.
+
+- If a redeem is not done at submission or the submission's initial redeem fails (for example, because the L2 gas price has increased unexpectedly), the submission fee is collected on L2 to cover the resources required to temporarily keep the ticket in memory for a fixed period (currently one week).
+At this point, _anyone_ can attempt to redeem the ticket again by calling `ArbRetryableTx`'s redeem precompile method, which donates the call's gas to the next attempt. 
+ArbOS will [enqueue the redeem][enqueue_link], which is its own special `ArbitrumRetryTx` type, to its list of redeems that ArbOS [guarantees to exhaust][exhaust_link] before moving on to the next non-redeem transaction in the block its forming. In this manner redeems are scheduled to happen as soon as possible, and will always be in the same block as the tx that scheduled it. 
+Note that the redeem attempt's gas comes from the call to redeem, so there's no chance the block's gas limit is reached before execution.
+If the fixed period (one week) elapses without a successful redeem, the ticket **expires** and will be [automatically **discarded**][discard_link], unless some party has paid a fee to [**renew**][renew_link] the ticket for another full period. A ticket can live indefinitely as long as it is renewed each time before it expires.
+
+[enqueue_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/block_processor.go#L245
+[exhaust_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/block_processor.go#L135
 [discard_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/retryables/retryable.go#L262
 [renew_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/retryables/retryable.go#L207
 
-## Retryable Ticket Lifecycle
 
-See [here](../assets/retryable_flowchart.png) for a flowchart of the lifecycle of a retryable ticket.
 
-### Submitting a Retryable
+:::caution Avoid Losing Funds!
+
+If a ticket expires after 7 days without being redeemed or re-scheduled to a future date, any value it carries will get lost without possibility of recovering them.
```

Comment:
Should that "any value" be "any value (other than the escrowed callvalue)" ?
It is mentioned in a few paragraphs below.

Reply:
I do like keeping the warning dire, but probably "could be lost" is more accurate. Eth in callvalue (as @TucksonDev said) or excess fees is recoverable, and, e.g., in theory a bridge could be constructed that allows for L1 recovery of failed deposits, etc.